dp/process-climo-means.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# process a model output file into climatological means
+# - copy input file to local temp
+# - process it with generate_climos.py
+# - copy results from local temp to permanent location
+# - delete copy of input file
+#
+# arguments:
+# $1: unique id for logfile so that this script can be run in several independent processes. Easy: Pass in $$.
+# $2: path to file to be processed
```

Comment:
It might be useful if the remaining arguments were a *list* of filenames to be processed. That way you can do the standard ``find -name *.nc | xargs process-climo-means.sh logfile.txt``.

To do this in the script, you would consume $1 as the logfile and then use the bash ``shift`` operation, and then the remaining args are available in $@ (I think).